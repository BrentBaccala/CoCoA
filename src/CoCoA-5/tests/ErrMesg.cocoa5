R ::= QQ[x,y,z];
K := NewFractionField(R);

Try
  PrintLn "gcd in field now throws Error 24May2012";
  gcd(zero(K), RingElem(K,"x")) = 1;
UponError e Do
  PrintLn GetErrMesg(e); PrintLn;
EndTry

Try
  PrintLn "gcd in field now throws Error 24May2012";
  gcd(zero(K), zero(K)) = 0;
UponError e Do
  PrintLn GetErrMesg(e); PrintLn;
EndTry
