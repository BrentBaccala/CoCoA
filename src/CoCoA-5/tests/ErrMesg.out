
# 
# 
# 
# 
Try # 
Try # 
Try # 
UponError # 
UponError # gcd in field now throws Error 24May2012
Ring is not a true GCD domain (e.g. it is a field)


# 
# 
Try # 
Try # 
Try # 
UponError # 
UponError # gcd in field now throws Error 24May2012
Ring is not a true GCD domain (e.g. it is a field)


# 